menu.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <section class="headline-container">
+            <div class="">
+                <img src="one-pass.svg" alt="one-pass">
+                <h1>Pass Bakery</h1>
+                <p>620 N 25th St, San Francisco, CA</p>
+                <h2 class="hours">Hours</h2>
+                <p>Mon-Fri 6AM-5PM PT,<br>Sat-Sun 7AM-4PM PT</p>
+            </div>
+            <div class="">
+                <img class="bread" src="bread.jpeg" alt="Gorgeous, fresh baked bread">
+            </div>
+        </section>
+        <h2>Coffee</h2>
+        <ul>
+            <li><h3>Espresso</h3> <br> <p class="menu-text">$2.75</p></li>
+            <li><h3>Cappuccino</h3> <br> <p class="menu-text">$3.50</p></li>
+            <li><h3>Latte</h3> <br> <p class="menu-text">$4.00</p></li>
+            <li><h3>Drip</h3> <br> <p class="menu-text">$2.50</p></li>
+            <li><h3>Pour Over</h3> <br> <p class="menu-text">$4.75</p></li>
+            <li class="invisible"></li>
```

Comment:
i would format these use an auto formatter, try downloading a plugin called `prettier` and configure it in your vscode settings:
1. `Cmd + ,`
2. search for format on save
3. check it, and then select formatter as something like `aspen.prettier`